demo/ncnn/cpp/README.md

```diff
@@ -63,7 +63,7 @@ YoloV5Focus      focus                    1 1 images 683
 ```
 
 ### Step5
-Use onnx_optimze to generate new param and bin:
+Use ncnn_optimze to generate new param and bin:
```

Comment:
Maybe update to `ncnnoptimize` or `ncnn_optimize` is more better.

Reply:
ok,tks

Reply:
/YOLOX/tools/train.py
Line42: expriment ->experiment. 
I know it's not a big deal maybe that's your way of abbreviating, but I just want you to know.
    parser.add_argument(
        "-f",
        "--exp_file",
        default=None,
        type=str,
        help="plz input your expriment description file",
    )